web/src/app/escalation-policies/PolicyStepForm.js

```diff
@@ -232,7 +232,7 @@ function PolicyStepForm(props) {
         </Grid>
         <Grid item xs={12}>
           <FormField
-            component={TextField}
+            component={NumberField}
             disabled={disabled}
             fullWidth
             label='Delay (minutes)'
```

Comment:
We should be able to remove `type` and `mapOnChangeValue` now, as they are handled within `NumberField`